atf_eregs/static/regulations/css/less/module/custom/full-footer.less

```diff
@@ -33,6 +33,14 @@
 		ul li:not(:first-child) {
 			text-align: right;
 		}
+        .footer-small ul li {
+			width: 20%;
+            white-space: nowrap;
+            text-size: 12px;
+        }
+        .footer-small ul li a.external{
+            font-size: 12px;
+        }
```

Comment:
This only affects non-wide screens. Was that the intent?

Also, the indentation is off quite a bit (which might have caused the confusion)